Tests/TestFuncs/TestFuncThrow/FixtureFuncThrow.swift

```diff
@@ -1,23 +1,15 @@
 import MockoloFramework
 
 
-let funcThrow2 = """
-import Foundation
-
-/// \(String.mockAnnotation)
-protocol FuncThrow {
-func update<T>(arg1: T, arg2: () throws -> T) rethrows -> T
-func update<T, U>(arg1: T, arg2: @escaping (U) throws -> ()) throws -> ((T) -> (U))
-}
-"""
-
```

Comment:
FYI

This test-case is not used and is covered by `funcThrow` test-case. So I deleted it.